.ci/install_runtime.sh

```diff
@@ -103,6 +104,11 @@ else
 	echo "Kata runtime will not set as a default in Docker"
 fi
 
+if [ "$MACHINETYPE" == "q35" ]; then
```

Comment:
I thought that defining MACHINETYPE=q35 was enough

Reply:
I think `-e` option should be added to the `make` command, for example here https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L74-L103